src/js/views/patients/patient/dashboard/add-workflow.scss

```diff
@@ -0,0 +1,39 @@
+.add-workflow__add-new {
+  font-style: italic;
+}
+
+.add-workflow__picklist {
+  .icon {
+    color: color(blue);
+  }
+
+  .is-highlighted .icon {
+    color: $white;
+  }
+}
+
+.add-workflow__no-results .icon {
+  color: $black-60;
+}
+
+.add-workflow__button {
+  padding-right: 36px;
+
+  .add-workflow__arrow {
+    left: auto;
+    right: 16px;
+  }
+}
+
+.add-workflow__picklist-item {
```

Comment:
I think we'll want to solve this problem generically for picklists

I think we can do this by adding a `picklist__item--icon` class and rather than wrapping the template in a span just set `itemClassName`

This will also require making sure picklists that current contain `icon`s don't break.